manifest/kubernetes/ubuntu/ncp-ubuntu.yaml

```diff
@@ -1077,10 +1078,6 @@ spec:
         - name: dir-tmp-usr-ovs-kmod-backup
           hostPath:
             path: /tmp/nsx_usr_ovs_kmod_backup
-
-        - name: host-usr-src
-          hostPath:
-            path: /usr/src
 {{else}}
```

Comment:
we can remove code line #L1081 - 1091.
{{else}}
..
#path: /tmp/nsx_user_ovs_kmod_backup.